.github/workflows/main.yml

```diff
@@ -4,9 +4,13 @@ on: [push]
 
 jobs:
   test:
-    runs-on: macos-latest
+    runs-on: macos-13-xl
     steps:
-    - uses: actions/checkout@master
+    - uses: actions/checkout@v4
+
+    - uses: maxim-lobanov/setup-xcode@v1
+      with:
+        xcode-version: latest-stable
```

Comment:
Now we have Embassy, is there still a reason to use the (rather outrageously spendy) bigger mac runner here?

Reply:
Good call! The non-xl image brings the test suite from ~5 minutes to ~8 minutes.